engine/src/main/java/org/hibernate/validator/internal/util/privilegedactions/GetDeclaredMethodHandle.java

```diff
@@ -37,7 +37,7 @@ public static GetDeclaredMethodHandle action(Lookup lookup, Class<?> clazz, Stri
 
 	/**
 	 * Before using this method on arbitrary classes, you need to check the {@code HibernateValidatorPermission.ACCESS_PRIVATE_MEMBERS}
-	 * permission against the security manager.
+-	 * permission against the security manager, if the calling class exposes the handle to clients.
```

Comment:
You put a dash at the beginning of the line to check I would really review it? ;)

Will fix.